src/System.Linq/tests/System.Linq.Tests.csproj

```diff
@@ -36,5 +37,10 @@
       <Name>System.Linq</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
```

Comment:
msbuild was happy with just entry in packages.config, but VS wasn't. I'll investigate.